legato/Component.cdef

```diff
@@ -0,0 +1,42 @@
+sources:
```

Comment:
Use lower case `component.cdef`.

Reply:
This is the default build name for legato build system.